packages/drivers/routerlicious-driver/src/urlUtils.ts

```diff
@@ -17,3 +19,23 @@ export const parseFluidUrl = (fluidUrl: string): URLParse => {
  */
 export const replaceDocumentIdInPath = (urlPath: string, documentId: string): string =>
     urlPath.split("/").slice(0, -1).concat([documentId]).join("/");
+
+export const createFluidUrl = (domain: string, pathname: string): string =>
+    "fluid://".concat(domain).concat(pathname);
```

Comment:
>.concat(domain).concat(pathname) [](http://example.com/codeflow?start=14&length=32)

(nit) why not .concat(domain, pathname)?

Reply:
Thanks. Changed it to .concat(domain, pathname)